src/main.rs

```diff
@@ -124,7 +125,7 @@ fn init_logger() {
 }
 
 fn get_book_dir(args: &ArgMatches) -> PathBuf {
-    if let Some(dir) = args.value_of("dir") {
+    if let Some(dir) = args.get_one::<String>("dir") {
```

Comment:
I don't think this is necessary to go through the roundtrip of making a String. I think it can do something like `get_one::<PathBuf>`, and then remove the call to `Path::new`.